.github/workflows/cut_release.yaml

```
@@ -0,0 +1,27 @@
+name: Create Tag for Release
+
+on:
+  push:
+    branches:
+    - 'master'
+    paths:
+     - 'chart/k8gb/Chart.yaml'
+
+jobs:
+  release:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Get tag
+        id: get_tag
+        run: |
+          new_tag=$(cat chart/k8gb/Chart.yaml | awk '/appVersion:/ {print $2}')
+          echo "::set-output name=new_tag::${new_tag}"
+      - name: Push Tag
+        uses: mathieudutour/github-tag-action@v5.6
+        if: "contains(github.event.head_commit.message, '[release]')"
```

Should we make it 
```suggestion
        if: "contains(github.event.head_commit.message, 'RELEASE:')"
```
to do it similar style as https://www.conventionalcommits.org/en/v1.0.0/ shared by @somaritane  ?